community/io/src/test/java/org/neo4j/io/pagecache/PageCacheTest.java

```diff
@@ -4416,7 +4416,7 @@ public void copyToHeapByteBufferFromReadPageCursorMustCheckBounds() throws Excep
     public void copyToDirectByteBufferFromReadPageCursorMustCheckBounds() throws Exception
     {
         configureStandardPageCache();
-        ByteBuffer buffer = ByteBuffer.allocateDirect( filePageSize );
+        ByteBuffer buffer = ByteBuffer.allocate( filePageSize );
```

Comment:
No, these three tests are explicitly about copying to/from _direct_ byte buffers.